examples/jenkins-master/jenkins-slave-template.json

```diff
@@ -0,0 +1,114 @@
+
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "s2i-to-jenkins-slave",
+    "annotations": {
+      "description": "Provide convenient way to convert any S2I builder to a Jenkins slave image",
```

Comment:
correct, the only requirement here is to have Fedora/CentOS/RHEL7 image to install Java. In case if RHEL7 you will have to run the builder on RHEL7 node that has subscriptions sorted out.